docs/core/index.md

```diff
@@ -38,10 +41,15 @@ You should see the following output:
 Hello World!
 ```
 
-## Support
+## Participate
 
-.NET Core is [supported by Microsoft](https://dotnet.microsoft.com/platform/support/policy), on Windows, macOS, and Linux. It's updated for security and quality several times a year, typically monthly.
+.NET Core is an open platform. Everyone is welcome to participate.
 
-.NET Core binary distributions are built and tested on Microsoft-maintained servers in Azure and supported just like any Microsoft product.
+* Product issues and questions should be filed as issues at [dotnet/core](https://github.com/dotnet/core/issues)
+* Contributions should be made on one of the project repos, such as [dotnet/runtime](https://github.com/dotnet/runtime), [dotnet/sdk](https://github.com/dotnet/sdk), [dotnet/rosyln](https://github.com/dotnet/roslyn), or [aspnetcore](https://github.com/dotnet/aspnetcore)
```

Comment:
Should we link here too: https://github.com/dotnet/core/blob/master/Documentation/core-repos.md?

```suggestion
* Product contributions should be made on one of the project repositories, such as [dotnet/runtime](https://github.com/dotnet/runtime), [dotnet/sdk](https://github.com/dotnet/sdk), [dotnet/rosyln](https://github.com/dotnet/roslyn), or [aspnetcore](https://github.com/dotnet/aspnetcore).
```